labellab-client/src/components/navbar/project.js

```diff
@@ -1,13 +1,54 @@
 import React, { Component } from 'react'
-import { Link } from 'react-router-dom'
-import { Image, Header, Dropdown, Icon } from 'semantic-ui-react'
+import { Link, Redirect } from 'react-router-dom'
+import _ from 'lodash'
+import { Image, Header, Dropdown, Icon, Search, Grid, Segment, Label } from 'semantic-ui-react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import { hasToken } from '../../utils/token'
 import { fetchUser } from '../../actions/index'
 import { TOKEN_TYPE } from '../../constants/index'
 import './css/navbar.css'
+
+const initialState = { isLoading: false, results: [], value: '' }
+
+const resultRenderer = ({ imageName, createdAt }) =>
+  <>
+    <Label content={imageName} />
+    <Label content={new Date(createdAt).toISOString().substring(0, 10)} />
+  </>
+
+
+resultRenderer.propTypes = {
+  title: PropTypes.string,
+  description: PropTypes.string,
+}
+
 class ProjectNavbar extends Component {
+  state = initialState
+
+  handleResultSelect = (e, { result }) => {
+    this.setState({ value: result.imageName })
+    window.location.href =
+      process.env.REACT_APP_HOST +
+      process.env.REACT_APP_SERVER_PORT +
+      '/project/' + this.props.project.projectId + '/images/#' + result._id
+  }
+
+  handleSearchChange = (e, { value }) => {
+    this.setState({ isLoading: true, value })
```

Comment:
Use prop destructuring as you used in render method.

Reply:
okay sure!